### ISSUE TITLE
Login hanging on localhost.

### ISSUE THREAD

**Issue by Deekor** [mkdynamic/omniauth-facebook#198]
I also posted this on StackOverflow: http://stackoverflow.com/questions/29759892/facebook-login-suddenly-hanging-in-development

My Facebook login has been working in the past but suddenly is hanging when I try to login from my localhost.

The last thing the server outputs is:

`[2015-04-20T16:52:50.509655 #77222]  INFO -- omniauth: (facebook) Callback phase initiated.`

it will sit there for several minutes (around 5) before finally logging in. In production on heroku this issue doesn't exist. Any ideas?

**Comment by comp615** [comment]
Mine has had this too, but to a lesser extent. I think it's honestly on the FB side :-/ even my requests after take several seconds

**Comment by simi** [comment]
@Deekor @comp615 can you inspect https://github.com/mkdynamic/omniauth-facebook/issues/98 and https://github.com/mkdynamic/omniauth-facebook/issues/31? It looks like similar issue.

TL;DR: disable IPv6

**Comment by comp615** [comment]
Will check it out. I do have IPv6 enabled I think at home. Also, this is all requests for me to FB (Koala Graph API included). Getting the JSON endpoint directly through ruby only takes a second as compared to 75 seconds via Koala if that helps any. :-/

Seems like this shouldn't be an issue though, any idea where in the stack this might be bugging out?

**Comment by simi** [comment]
Hello @comp615 @Deekor. Any news here? I have no idea what can be wrong in `omniauth-facebook` codebase causing this weird behaviour. :(

**Comment by mkdynamic** [comment]
Closing this, seems specific to environment/setup and not a bug. Please update/re-open if you have more info.

**Comment by mkdynamic** [comment]
Closing this, seems specific to environment/setup and not a bug. Please update/re-open if you have more info.
